app/screens/SignUp.java

```diff
@@ -27,18 +26,52 @@ public void display() {
 
             // handles exceptions
             try {
+
                 User user = auth.signUp(username, password);
 
-                System.out.print("Enter your store name: ");
-                String storeName = sc.nextLine();
+                System.out.println();
+                MenuTitle.displayMainTitle();
+                MenuTitle.displaySubTitle("[Set-up]");
+
+                String storeName = null;
+                while (storeName == null || storeName.isEmpty()) {
```

Comment:
here you can just set storeName = """, and remove the first conditional statement of the while loop

Reply:
okiee